3ds Max/Max2Babylon/Tools/Tools.cs

```diff
@@ -1130,6 +1131,70 @@ public static string GetRelativePath(string fromPath, string toPath)
             return relativePath;
         }
 
+
+        public static string UnformatPath(string formattedPath)
+        {
+            string newPath = formattedPath;
+            newPath = Regex.Replace(formattedPath, @"[()]", string.Empty);
```

Comment:
I'm not entirely sure what your usecase is here.

I was assuming that this allows us to use relative pathing shortcuts like ./ ../ and ~/ when indicating the output directory for the babylon exporter, and we would be able to resolve that to the full path on export but this doesn't appear to do that.

Can you please give me some context behind these changes and why they're useful? It would be especially great if I can get some testing scenarios.

Reply:
My idea is that we should try, when possible, to store the path relative to the project folder,
but at the same time always display the absolute bath on the user who grabs the file
That is the reason why I added the format/unformat methods to display the project folder part of the path around parenthesis
something like this
user choice:
c:\my_max_project\assets\3d\gltf\model.gltf
we display
(c:\my_max_project)\assets\3d\gltf\model.gltf
we store
\assets\3d\gltf\model.gltf

user B open the max file with the project folder configured to other location
we display
(d:\3ds_max_projects\my_max_project)\assets\3d\gltf\model.gltf